dapps/evancore.vue.libs/src/components/profile-picture/profile-picture.scss

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Should we move this to ui-core general css style? So it could be used by other frameworks.